warehouse/templates/index.html

```diff
@@ -85,34 +90,30 @@ <h1 class="homepage-banner__title">Find, install and publish Python packages wit
 </section>
 
 <section class="horizontal-section horizontal-section--grey">
-  <div class="site-container">
-    <div class="col-half">
+  <div class="site-container" >
+    <div class="col-half" data-controller="show-trending-projects">
```

Comment:
you could simply use `data-controller="show-more"` here and below, Stimulus will create separate instances